deployments/k8s/helm/signalfx-agent/values.yaml

```diff
@@ -83,13 +83,17 @@ traceEndpointUrl:
 # The SignalFx API base URL. (default: "https://api.signalfx.com")
 apiUrl:
 
-# clusterName must be provided.  It is an arbitrary value that identifies this
-# K8s cluster in SignalFx.  This will be the value of the 'kubernetes_cluster'
-# dimension on every metric sent by the agent.  It will also be the value of
-# the `cluster` config option that is used to set a `cluster` property on the
-# `kubernetes_node` dimension.
+# An arbitrary value that identifies this K8s cluster in SignalFx.  This will be
+# the value of the 'kubernetes_cluster' dimension on every metric sent by the
+# agent (unless overriden by `kubernetesClusterName`).  It will also be the
+# value of the `cluster` config option that is used to set a `cluster` property
+# on the `kubernetes_node` dimension. (REQUIRED)
```

Comment:
This is a good explanation of _how_ this works but doesn't really explain _why_ one would need this. I think this is going to be confusing for users who are unfamiliar with the need for the `cluster` property.
We should add something about setting this value to the same value as your Smart Gateway (or gateway cluster?).
Better to be explicit in this case I think.

Reply:
Yeah should probably mention the Smart Gateway aspect.